examples/mysql-kopia-repository-server/README.md

```diff
@@ -0,0 +1,423 @@
+# MySQL
+
+[MySQL](https://MySQL.org) is one of the most popular database servers in the world. Notable users include Wikipedia, Facebook and Google.
+
+## Introduction
+
+This chart bootstraps a single node MySQL deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.20+
+- PV provisioner support in the underlying infrastructure
+- Kanister controller version 0.90.0 installed in your cluster, let's assume in Namespace `kanister`
+- Kanctl CLI installed (https://docs.kanister.io/tooling.html#install-the-tools)
+
+## Installing the Chart
+
+To install the MySQL database using the `bitnami` chart with the release name `mysql-release`:
+
+```bash
+# Add bitnami in your local chart repository
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+
+# Update your local chart repository
+$ helm repo update
+
+# Install the MySQL database
+
+> **Note**: The custom image for MySQL is required in order to utilize the Kopia Repository Server based functions.
+
+$ helm install mysql-release bitnami/mysql \
+  --namespace mysql \
+  --create-namespace \
+  --set image.repository=kanisterio/mysql-kopia \
+  --set image.tag=0.90.1 \
+  --set auth.rootPassword='<mysql-root-password>'
+```
+
+The command deploys a MySQL instance in the `mysql` namespace.
+
+By default, a random password will be generated for the root user. For setting your own password, use the `auth.rootPassword` param as shown above.
+
+You can retrieve your root password by running the following command. Make sure to replace [YOUR_RELEASE_NAME] and [YOUR_NAMESPACE]:
+
+    `kubectl get secret [YOUR_RELEASE_NAME] --namespace [YOUR_NAMESPACE] -o jsonpath="{.data.mysql-root-password}" | base64 --decode`
+
+> **Tip**: List all releases using `helm list --all-namespaces`, using Helm Version 3.
+
+## Integrating with Kanister
+
+If you have deployed MySQL application with name other than `mysql-release` and namespace other than `mysql`, you need to modify the commands(backup, restore and delete) used below to use the correct release name and namespace
+
+### Create Repository Server CR
+
+Create Kopia Repository Server CR if not created already
+
+- Create Kopia Repository
+```bash
+$ kopia --log-level=error --config-file=/tmp/kopia-repository.config \
+--log-dir=/tmp/kopia-cache repository create --no-check-for-updates \
+--cache-directory=/tmp/cache.dir --content-cache-size-mb=0 \
+--metadata-cache-size-mb=500 --override-hostname=mysql.app \
+--override-username=kanisterAdmin s3 --bucket=<s3_bucket_name> \
+--prefix=/repo-controller/ --region=<s3_bucket_region> \
+--access-key=<aws_access_key> --secret-access-key=<aws_secret_access_key> --password=<repository_password>
+```
+
+- Create Certificates and Location Secrets for Kopia Repository Server
+```bash
+$ openssl req -newkey rsa:2048 -nodes -keyout key.pem -x509 -days 365 -out certificate.pem
+```
```

Comment:
It looks like we generated the cert here and then are creating the secret later. This is not very straight forward. Let's create each secret separately. 
For example, generate cert and immediately after that create secret. 
Then talk about the other secret yaml and then create that secret.

Reply:
Done !